RUFAS/routines/field/soil/soil_erosion.py

```diff
@@ -168,9 +169,20 @@ def _determine_clay_silt_ratio_factor(silt_fraction: float, clay_fraction: float
         SWAT Theoretical documentation eqn. 4:1.1.7
 
         """
+        om = OutputManager()
         if silt_fraction == 0 and clay_fraction == 0:
-            raise ValueError("Cannot have fractions of silt and clay in the soil both be 0")
-        return (silt_fraction / ((clay_fraction) + (silt_fraction))) ** 0.3
+            info_map = {
+                "class": SoilErosion.__name__,
+                "function": SoilErosion._determine_clay_silt_ratio_factor.__name__,
+            }
+            om.add_warning(
+                "Silt and clay fractions are 0",
+                "Both silt and clay fractions are 0 which will lead to unreliable predictions of erosion and soil"
+                " emissions",
```

Comment:
When raising a warning or error it is good to not just list the cause but also the effect. For example:
```suggestion
                "Silt and clay fractions in the soil are 0, which will lead to unreliable predictions of erosion and soil"
                " emissions",
                "It is assumed that the ratio of clay to silt in the soil layer will not have any effect on the amount of erosion from the soil.",
```

Reply:
Again, I'm a little mixed about this since the user should know what they're doing at this point, but adding details is never bad.